tests/js/t9410.nim

```diff
@@ -0,0 +1,234 @@
+discard """
+  output: '''3 3
```

Comment:
Please use `doAssert` instead of `echo` + expected output.

Reply:
Will do, I used `echo`s since it initially made it easier to just compare the outputs of JS and C at a glance, but I guess the test grew a bit too large for that now.